osf/models/node.py

```diff
@@ -1673,7 +1679,12 @@ def fork_node(self, auth, title=None, parent=None):
         forked.save()
 
         # Need to call this after save for the notifications to be created with the _primary_key
-        project_signals.contributor_added.send(forked, contributor=user, auth=auth, email_template='false')
```

Comment:
@Johnetordoff apparently I changed the business logic here, but it seems initially wrong, forked contributors shouldn't be notified?  Check if we are doubling up notifications.